Example/SwipePopViewController/ViewController.swift

```diff
@@ -31,7 +31,7 @@ class ViewController: UIViewController {
         
         switch mode {
         case .native:
-            applyNativePopGesture()
+            addSwipePopGesture()
```

Comment:
Please restore the old native code (`applyNativePopGesture` should not be changed)